lib/browser/api/module-list.js

```diff
@@ -8,6 +8,7 @@ module.exports = [
   {name: 'dialog', file: 'dialog'},
   {name: 'globalShortcut', file: 'global-shortcut'},
   {name: 'ipcMain', file: 'ipc-main'},
+  {name: 'inAppPurchase', file: 'in-app-purchase'},
```

Comment:
It also should be available on macOS only.

Reply:
What should I do to restrict this to macOS only ?

Reply:
Oh, there's not existing way to do it. I guess I or somebody else can add it after this PR is merged.
Let's skip it for now.